packages/react-charting/src/utilities/SVGTooltipText.tsx

```diff
@@ -41,6 +41,16 @@ interface ISVGTooltipTextProps {
    */
   maxHeight?: number;
 
+  /**
+   * Pass false to make the data not focusable, though keyboard. true by default
+   */
+  shouldReceiveFocus?: boolean;
```

Comment:
Add a comment about the scenario when shouldReceiveFocus should be set to false.

Reply:
repetition of the same datapoint for narrator.

Reply:
This is a bug in lot of charts, it's being tracked here -> https://uifabric.visualstudio.com/iss/_workitems/edit/9036

Reply:
Could you add this bug id and the issue in the comment.

Reply:
### Issue
The Screen Reader reads the aria-label of the svg repeatedly for each of the child elements of the svg. 
For eg: 
<svg aria-label="Pie Chart">
<g aria-label="a">
<g aria-label="b">
<g aria-label="c">
</svg>

for all the 3 <g> tags, the screen reader will pronounce the aria-label of the <svg>

Tracking link : https://uifabric.visualstudio.com/iss/_workitems/edit/9036